global-forms.json

```diff
@@ -14868,6 +14934,66 @@
                   "fontStyle": "bold",
                   "fontColour": "rgba(255,0,0,1)"
                 }
+              },
+              {
+                "id": "addSection",
+                "label": "Add Section",
+                "margin": {
+                  "top": "8",
+                  "right": "0",
+                  "bottom": "8",
+                  "left": "0"
+                },
+                "padding": {
+                  "top": "8",
+                  "right": "8",
+                  "bottom": "8",
+                  "left": "8"
+                },
+                "background": {
+                  "background_color": "rgba(18, 110, 0,1)"
```

Comment:
[Optional] I think that having the same color for all the themes makes no sense as they usually have main base color.  Our themes look terrible, but best to keep it consistent. We can just use the same background color from the theme instead of using the same one - when doing this repeating sections will not stick out.

ping @paolobueno

Reply:
I honestly doubt anyone is using our themes in production :D